SPECS/kata-containers-cc/mariner-coco-build-uvm.sh

```diff
@@ -43,5 +43,5 @@ sed -i 's/@BINDIR@\/@AGENT_NAME@/\/usr\/bin\/kata-agent/g'  ${ROOTFS_DIR}/usr/li
 # build image
 pushd ${OSBUILDER_DIR}
 mv rootfs-builder/rootfs-cbl-mariner cbl-mariner_rootfs
-sudo -E PATH=$PATH make DISTRO=cbl-mariner MEASURED_ROOTFS=yes DM_VERITY_FORMAT=kernelinit image
+sudo -E PATH=$PATH make DISTRO=cbl-mariner MEASURED_ROOTFS=yes DM_VERITY_FORMAT=kernelinit OS_VERSION=3.0 image
```

Comment:
should not be necessary for image and initrd. reconfirming and removing if true

Reply:
Confirmed that it's not necessary for image and initrd:
https://dev.azure.com/mariner-org/mariner/_build/results?buildId=602725&view=logs&j=246943f6-b1af-5ad0-0a7e-380cdc2643ed&t=40a501d7-8089-55dd-5041-5695c7600910&l=131

Reply:
Okay, dropped the OS_VERSION from image,initrd make calls.